Tween/ParalleledAnimation.cs

```diff
@@ -9,21 +11,29 @@ public class ParalleledAnimation : AnimationCollection
 	{
 		protected override void PlayQueuedAnimations()
 		{
-			Animations.RemoveAll(animation => !animation.IsValid);
-
-			if (Animations.Count > 0)
+			for (var i = Animations.Count - 1; i >= 0; --i)
 			{
-				Animations.ForEach(animation => animation.Play(HandleEachAnimationCompleted, () =>
+				var animation = Animations[i];
+				if (animation.IsValid)
 				{
-					if (!animation.IsValid)
+					animation.Play(HandleEachAnimationCompleted, () =>
 					{
-						Animations.Remove(animation);
-					}
-				}));
+						if (IsPlaying && !animation.IsValid)
```

Comment:
Do we want to check this `!animation.IsValid`? I could be mistaken but I think the only time we would get this callback is if it's not valid anyway, or the overall collection `IsPlaying` was false due to being aborted.

Plus if for some reason an animation was aborted while valid, it would never be removed from the list and the overall AnimationCollection couldn't complete (same for SequencedAnimation, because of `if (++NumberOfAnimationsCompleted == Animations.Count)`). I know we're not trying to cover the individual modification of animations while in a collection, but if the above it true there's no point making it hang for nothing.

Reply:
This is `onAbort` callback. Handling if the aborted animation inside is invalid or not.

This is specifically designed for animation collection. If you have a collection inside a collection, and that collection went invalid and aborted, this parent collection will continue to work...

Reply:
And don't worry -- as I mentioned all the error handlings will get a refactory.

I just left it as it is. It's not breaking anything at least...

Reply:
Hadn't realised the case with collections within collections was different, you're right 👍